packages/office-ui-fabric-react/src/components/DatePicker/examples/DatePicker.Bounded.Example.tsx

```diff
@@ -1,6 +1,6 @@
 import * as React from 'react';
 import { DatePicker, DayOfWeek, IDatePickerStrings } from 'office-ui-fabric-react/lib/DatePicker';
-import { addMonths, addYears } from '../../../utilities/dateMath/DateMath';
+import { addMonths, addYears } from 'office-ui-fabric-react/lib/Utilities/dateMath/DateMath';
```

Comment:
I'm worried about this because isn't our guidance to not import beyond the root? This is a potentially fragile deep import that is making it into official example

Reply:
this might be /Utilities. I'm not sure. i'll check.